editor.planx.uk/src/@planx/components/Send/Editor.tsx

```diff
@@ -81,44 +69,7 @@ const SendComponent: React.FC<Props> = (props) => {
         newCheckedValues = [...formik.values.destinations, value];
       }
 
-      formik.setFieldValue(
-        "destinations",
-        newCheckedValues.sort((a, b) => {
-          const originalValues = options.map((cb) => cb.value);
-          return originalValues.indexOf(a) - originalValues.indexOf(b);
-        }),
-      );
-
-      // Show warnings on selection of BOPS or Uniform for likely unsupported services
-      //   Don't actually restrict selection because flowSlug matching is imperfect for some valid test cases
-      const teamSlug = window.location.pathname?.split("/")?.[1];
-      const flowSlug = window.location.pathname?.split("/")?.[2];
-      if (value === "bops" && newCheckedValues.includes(value)) {
-        alert(
-          "BOPS only accepts Lawful Development Certificate, Prior Approval, and Planning Permission submissions. Please do not select if you're building another type of submission service!",
-        );
-      }
-
-      if (
-        value === "uniform" &&
-        newCheckedValues.includes(value) &&
-        flowSlug !== "apply-for-a-lawful-development-certificate" &&
-        !["buckinghamshire", "lambeth", "southwark"].includes(teamSlug)
-      ) {
-        alert(
-          "Uniform is only enabled for Bucks, Lambeth and Southwark to accept Lawful Development Certificate submissions. Please do not select if you're building another type of submission service!",
-        );
-      }
-
-      if (
-        value === "s3" &&
-        newCheckedValues.includes(value) &&
-        !["barnet", "southwark"].includes(teamSlug)
-      ) {
-        alert(
-          "AWS S3 uploads require API tokens and are currently being prototyped with select councils. Reach out on Slack #planx-alternatives-to-integrations before selecting this option please.",
-        );
-      }
```

Comment:
No more `alerts` on selection ! Instead will either logically `disable` Uniform switch or shift these types of notifications to future on-publish validation checks.